src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -905,6 +1036,18 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
           renderSeatingNumberInput()}
 
         {renderRadioToggle()}
+
+        <UserStatusManager
+          venueId={venueId}
+          checked={hasUserStatuses}
+          userStatuses={userStatuses}
+          onCheck={toggleUserStatus}
+          onDelete={removeUserStatus}
+          onAdd={addUserStatus}
+          onPickColor={updateStatusColor}
+          onChangeInput={updateStatusText}
+        />
```

Comment:
as we decide to show sttuses by default we should use `DEFAULT_SHOW_USER_STATUSES`(from `settings`) somehow here as well.

Reply:
Just ship guys! We are in a bit of a crisis here. These don't seem like feedback around potential bugs but just styling issues.

Reply:
@jonboutelle Please refrain from jumping in and commenting on situations you don't really understand. Having a panicked "omg just ship now stop spending so much time zomg" isn't helpful to anyone.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/6db8a73718475d1f8e780610acf3becd6f5e4c3b